crates/utils/Cargo.toml

```diff
@@ -77,9 +77,8 @@ deser-hjson = { version = "2.2.4", optional = true }
 smart-default = { version = "0.7.1", optional = true }
 lettre = { version = "0.11.10", default-features = false, features = [
   "builder",
-  "tokio1",
```

Comment:
@Nutomic it did find one here, and also on L71 removing serde.

It took like an hour to run, and also created a `target` dir of > 100GB lol, but at least it did find a few things.

Reply:
Better than nothing, but this definitely wont have any noticable effect on build time.